resources/pvc.py

```diff
@@ -71,21 +71,21 @@ def resize_pvc(self, new_size, verify=False):
         return True
 
 
-def delete_all_pvcs():
+def delete_pvcs(pvc_name):
     """
-    Deletes all pvc in namespace
+    Deletes specific pvc in namespace
+
+    Args:
+        pvc_name (str): Name of pvc for deletion
 
     Returns:
         bool: True if deletion is successful
     """
     ocp_pvc_obj = OCP(
         kind=constants.PVC, namespace=config.ENV_DATA['cluster_namespace']
     )
-    ocp_pvc_list = get_all_pvcs()
-    pvc_list = ocp_pvc_list['items']
-    for item in pvc_list:
-        ocp_pvc_obj.delete(resource_name=item.get('metadata').get('name'))
-
+    log.info(f"Deleting PersistentVolumeClaim with name {pvc_name}")
```

Comment:
Can you format your logging messages like in the coding guidelines please? https://github.com/red-hat-storage/ocs-ci/blob/master/docs/coding_guidelines.md

Unless I'm missing something that isn't going to render the value of ``pvc_name`` in that log message will it?

Reply:
Done